ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-devel__downstream-candidate.yaml

```diff
@@ -25,7 +25,7 @@ tests:
     cluster_profile: azure-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
-      CATALOG_SOURCE_IMAGE: quay.io/redhat-user-workloads/ose-osc-tenant/osc-test-fbc:osc-test-fbc-on-push-pzw5x-build-image-index
+      CATALOG_SOURCE_IMAGE: quay.io/redhat-user-workloads/ose-osc-tenant/osc-test-fbc:1.10.0-1753281667
```

Comment:
This doesn't seem related to CoCo, could you please add it in a separate commit (and is it really required at all? Originally I expected those fields not to be updated regularly as it should serve as a base for the testing PR that should define all the sources/images/...)